packages/next-common/context/connectedWallet/index.js

```diff
@@ -0,0 +1,34 @@
+import { CACHE_KEY } from "next-common/utils/constants";
+import { createContext, useContext, useEffect, useState } from "react";
+
+const ConnectedWalletContext = createContext(null);
+
+export default ConnectedWalletContext;
+
+export function ConnectedWalletProvider({ children }) {
+  const [connectedWallet, setConnectedWallet] = useState();
+
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      setConnectedWallet(localStorage[CACHE_KEY.lastLoginAddress]);
```

Comment:
Why not keep the connect wallet(polkadot extension, metamask, talisman, etc) too in this context?